### ISSUE TITLE
Horse Archer Description: "Unlike the Chariot Archer, the Horse Archer does not require Horses".

### ISSUE THREAD

**Issue by Alsark** [LoneGazebo/Community-Patch-DLL#2612]
_1. Mod version (i.e Date - 4/23):_
9/30 beta version

_2. Mod list (if using Vox Populi only, leave blank):_
Vox Populi + EUI

_3. Error description:_ 
The Horse Archer description mentions that "unlike the Chariot Archer, the Horse Archer does not require Horses", implying that it's a Chariot Archer upgrade rather than a Skirmisher upgrade. The description also doesn't mention the strength boost over a normal Skirmisher.

![image](https://cloud.githubusercontent.com/assets/22311638/19215817/64b15094-8d76-11e6-8c76-fae1eaad9fe4.png)

I'm noticing some description errors in other units, as well. If it would help, I wouldn't mind volunteering to update and upload some description changes. They're in Community Balance Overhaul\Balance Changes\Text\en_US\CivText.sql, right?

**Comment by LoneGazebo** [comment]
They're in all the .sql text files (unittext, civtext, etc.). Any help would be great!

**Comment by Alsark** [comment]
It's the least I can do for this great mod you and the rest of the team have provided to the community.
I'll get on that and have something ready by the end of next week.

**Comment by LoneGazebo** [comment]
Thanks!

**Comment by Alsark** [comment]
Okay so for anything not in CivText.sql, I'm not having much luck finding. UnitText has some of the "Game Info" entries, but not Strategy entries for the unique units.

I've attached CivText.sql (which I had to rename as CivText.txt to upload), as well as a change log of what I have so far.

[Unit Description Change Log.xlsx](https://github.com/LoneGazebo/Community-Patch-DLL/files/518105/Unit.Description.Change.Log.xlsx)

[CivText.txt](https://github.com/LoneGazebo/Community-Patch-DLL/files/518106/CivText.txt)

**Comment by LoneGazebo** [comment]
Well not every entry is there, as not all are edited. non-edited entries will have to be grabbed from the actual game files and edited as I do them in the SQL.

**Comment by Alsark** [comment]
Perfect - thanks! Here's all of the updated descriptions. Additionally, I incorporated your 10/8 changes into my CivText so it'll be ready to slip into the next version. 

The only other minor thing I noticed, but didn't mess with, was that the Portuguese Nau says the Exotic Cargo ability can be used on Civilizations or City-States. The Exotic Cargo promotion only mentions City-States. So one or the other description could use a minor text tweak, but I've not played with Portugal to know which.

[CivText.txt](https://github.com/LoneGazebo/Community-Patch-DLL/files/518164/CivText.txt) (CivText.sql)
[UnitText.txt](https://github.com/LoneGazebo/Community-Patch-DLL/files/518149/UnitText.txt) (UnitText.xml)
[Unit Description Change Log.xlsx](https://github.com/LoneGazebo/Community-Patch-DLL/files/518150/Unit.Description.Change.Log.xlsx)

I hope this helps! Let me know if you'd prefer any of these files in a different manner.
